src/app/shared/components/header/header.component.html

```diff
@@ -4,7 +4,7 @@
     <h1>VIDEO COURSE</h1>
   </a>
   <div class="user-info" *ngIf="authenticationService.isAuthenticated">
-    <span>{{authenticationService.currentUser.firstName}}</span>
+    <span>{{authenticationService.currentUser?.name.first}} {{authenticationService.currentUser?.name.last}}</span>
```

Comment:
Don't get data from service in template, use component for it. 
Also check this component I see errors in console.

Reply:
Done